man/man5/zpool-features.5

```diff
@@ -884,5 +884,25 @@ The feature will only return back to being \fBenabled\fR when the pool
 is rewound or the checkpoint has been discarded.
 .RE
 
+.sp
+.ne 2
+.na
+\fB\fBlivelist\fR\fR
+.ad
+.RS 4n
+.TS
+l l .
+GUID	com.delphix:livelist
+READ\-ONLY COMPATIBLE	yes
+DEPENDENCIES	none
+.TE
+
+This feature allows clones to be deleted faster than the traditional method
+when there are large number of random/sparse writes made to the clone.
```

Comment:
How about: "This feature allows clones to be deleted faster than the traditional method
when a large number of random/sparse writes have been made to the clone."